src/Marker.tsx

```diff
@@ -14,6 +15,7 @@ export interface MarkerProps extends H.map.Marker.Options, H.geo.IPoint {
   bitmap?: string;
   data?: any;
   draggable?: boolean;
+  children?: any;
```

Comment:
Just a really minor thing please use correct type here

Reply:
I added the type React.ReactElement<any>